invenio_records_resources/schemas/url_args.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Marshmallow JSON schema."""
+
+
+from marshmallow import Schema, ValidationError, fields, post_load, validate, \
+    validates_schema
+
+from ..pagination import PagedIndexes
+
+# TODO: Make configurable
+DEFAULT_RESULTS_PER_PAGE = 25
+DEFAULT_MAX_RESULTS = 10000
+
+
+class PaginationURLArgsSchemaV1(Schema):
+    """Schema for pagination URL args."""
+
+    page = fields.Int(validate=validate.Range(min=1), missing=1)
```

Comment:
I have the feeling we are sort of re-implementing the arguments parsing of `flask-resources`. Is it impossible to re-use it? or modify it to make it configurable?

Reply:
Ah, I "moved" the functionality here. This PR: https://github.com/inveniosoftware/flask-resources/pull/74 gets rid of the leftovers in flask-resources (but `paginate.py` wasn't used in flask-resources since early this month).

We have to move it, because the service needs to be responsible for pagination, so that it can generate pagination links outside of a request-response (a view).

Reply:
Im still not convinced since pagination is something that most REST apis need in a collection resource. But okay, if blocking just merge we tackle it later on.